tests/acceptance/acceptancesuite.js

```diff
@@ -3,17 +3,23 @@ import VerticalPage from './pageobjects/verticalpage';
 import { setupServer, shutdownServer } from './server';
 import FacetsPage from './pageobjects/facetspage';
```

Comment:
This is likely out of scope for your item, but I think our acceptance tests need some TLC. At the very least, we should break them out into multiple files (maybe one for vertical, universal, facets, and storage). This file is getting pretty bulky. But, we can make a separate item for that.